services/ymax-planner/src/watchers/cctp-watcher.ts

```diff
@@ -110,3 +131,50 @@ export const watchCctpTransfer = ({
     }, timeoutMs);
   });
 };
+
+export const lookBackCcctp = async ({
```

Comment:
Too many `c`s :wink:
```suggestion
export const lookBackCctp = async ({
```